python/core/__init__.py

```diff
@@ -139,7 +139,12 @@ def __hash__(self):
     NULL = QPyNullVariant(int)
 
 except ImportError:
-    pass
+    try:
+        # TODO: Fixme, this creates an invalid variant, not a NULL one
```

Comment:
Ok. Wasn't sure if it was a generic "sometime later" to-do or an overlooked pre-merge one.